packages/create-sitecore-jss/src/common/utils/helpers.ts

```diff
@@ -26,32 +26,30 @@ export const getPascalCaseName = (name: string): string => {
 };
 
 /**
- * Provides `package.json` data
- * @param {string} [pkgPath] path to `package.json`. Default is './package.json'.
- * @returns `package.json` data
+ * Provides json data from a file
+ * @param {string} [jsonFilePath] path to the .json file.
```

Comment:
Remove brackets here (no longer optional)
```suggestion
 * @param {string} jsonFilePath path to the .json file.
```